test/parallel/test-fs-promises-file-handle-append-file.js

```diff
@@ -39,6 +39,24 @@ async function validateAppendString() {
   await fileHandle.close();
 }
 
-validateAppendBuffer()
-  .then(validateAppendString)
-  .then(common.mustCall());
+async function doAppendAndCancel() {
+  const filePathForHandle = path.resolve(tmpDir, 'dogs-running.txt');
+  const fileHandle = await open(filePathForHandle, 'w+');
+  try {
+    const buffer = Buffer.from('dogs running'.repeat(512 * 1024), 'utf8');
+    const controller = new AbortController();
+    const { signal } = controller;
+    process.nextTick(() => controller.abort());
+    await assert.rejects(fileHandle.appendFile(buffer, { signal }), {
+      name: 'AbortError'
+    });
+  } finally {
```

Comment:
The `try...catch` is not needed as the process exits when an error occurs.

Reply:
Thanks! I removed in [78c5b07](https://github.com/nodejs/node/pull/56972/commits/78c5b076821c24b8ce7ff6d912e28d45138066b9).